src/components/QuestionCard/styled.js

```diff
@@ -28,6 +26,14 @@ const StyledQuestionComponent = styled.section`
 		top: 50px;
 		right: 10px;
 	}
+
+	${props =>
+		props.ownedQuestion
+			? css`
+					background-color: var(--lightblue);
+					color: var(--dark);
+			  `
+			: null}
 `;
```

Comment:
You could use the logical AND (`&&`) here instead. 
```suggestion
	${props =>
		props.ownedQuestion
			&& css`
					background-color: var(--lightblue);
					color: var(--dark);
			  `}
`;
```

Reply:
Thanks for the suggestion! This is indeed an unnecessary return value :)